tests/e2e/integration_suite_test.go

```diff
@@ -17,14 +17,14 @@
 
 package deploy_test
 
-import (
-	"testing"
+// import (
```

Comment:
I haven't has the time to do them yet, we have to rewrote them completely because the deploy command is changed and testenv cannot be used with the watch status capability turned on